packages/vanilla-i18n/src/translationStore.ts

```diff
@@ -1,3 +1,5 @@
+import { getMeta } from "@vanilla/library/src/scripts/utility/appUtils";
```

Comment:
This import should be removed. See below.

Reply:
done